tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -146,17 +146,10 @@ class DatasetParams {
   // Returns the functions that will be used when running the dataset op.
   virtual std::vector<FunctionDef> func_lib() const { return {}; }
 
-  // Returns a function definition for the transformation performed by
-  // this type of dataset.
-  virtual Status CreateFactory(FunctionDef* fdef) const {
-    return errors::Unimplemented(
-        "CreateFactory is not implemented for params '", op_name(), "'");
-  }
```

Comment:
`CreateFactory(...)` is not needed now as all kinds of `DatasetParams` are supported as the inputs in the pipeline (in theory).